lib/modules/platform/azure/__snapshots__/index.spec.ts.snap

```diff
@@ -30,10 +30,10 @@ exports[`modules/platform/azure/index createPr() should create and return a PR o
 }
 `;
 
-exports[`modules/platform/azure/index createPr() should create and return a PR object with auto-complete set 1`] = `
+exports[`modules/platform/azure/index createPr() when usePlatformAutomerge is set should create and return a PR object with auto-complete set 1`] = `
 {
   "autoCompleteSetBy": {
-    "id": 123,
+    "id": "123",
```

Comment:
why does this change?

Reply:
The response object from `azureApi.gitApi.updatePullRequest` is spread onto the return value from `azure.createPr`.  `id` is `string` on both `createdBy` and `autoCompleteSetBy`, which was revealed as a type error once I converted these tests to `partial<IGitApi>`.  It's not actually declared on the interface `createPr` returns, but it did affect the snapshot.